### PR TITLE
Fixed links in docs/using.rst

### PR DESCRIPTION
**Pull request by nurav**
Fixed the dysfunctional links in using.rst. @containscafeine

### CODE REVIEW

File: docs/using.rst [hunk 1]
```diff
@@ -185,7 +184,7 @@ expected output is::
     127.0.0.1   kubernetes.io/hostname=127.0.0.1   Ready
 
 .. _helloapache: https://registry.hub.docker.com/u/projectatomic/helloapache/
-.. _README: ../components/centos/centos-k8s-singlenode-setup/README.rst
+
 .. _Vagrantfile: ../components/centos/centos-k8s-singlenode-setup/Vagrantfile
```
Comment (bexelbie) [inline docs/using.rst]: Shouldn't this line also be deleted?

**Comment by praveenkumar** [comment]
#build

**Comment by bexelbie** [comment]
LGTM

**Comment by LalatenduMohanty** [comment]
LGTM

**Pull request merged**